testrunner-gradle-plugin/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Yeah, I think we'd better keep this file.

As specified in [gradle doc](https://docs.gradle.org/current/userguide/gradle_wrapper.html), gradlew (Gradle Wrapper) is a script that invokes a declared version of Gradle, downloading it beforehand if necessary. As a result, developers can get up and running with a Gradle project quickly **without having to follow manual installation processes saving your company time and money**. It helps solve incompatible gradle version issues as it is possible that the developer's local gradle version cannot build this repo successfully.

Reply:
The link you sent seems to be about projects using Gradle to build, not about building a Gradle plugin. Can you build your Gradle plugin using Maven? If so, you should be able to remove all of these Gradle files. 

Looking at NonDex, it seems they use Maven to build everything but the Gradle plugin. This means that to build the NonDex Maven plugin, they have to use Maven, and then to build the Gradle plugin they have to use Gradle. It would be helpful if confirm whether Maven can just build both plugins.

Reply:
Thank you so much for your suggestion! I surprisingly find this is possible: https://stackoverflow.com/questions/27555560/how-can-i-build-a-gradle-plugin-with-maven.

The new changes are updated. I have tested it and now maven can build gradle plugin successfully! Simply, users can run "mvn clean install" in root directory, and then the plugin will be available in local maven repo (~/.m2/repository).